app/uk/gov/hmrc/customsexportsinternalfrontend/config/AppConfig.scala

```diff
@@ -16,22 +16,32 @@
 
 package uk.gov.hmrc.customsexportsinternalfrontend.config
 
-import javax.inject.{Inject, Singleton}
-import play.api.Configuration
+import javax.inject.{Inject, Named, Singleton}
+import play.api.{Configuration, Environment, Mode}
 import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 @Singleton
-class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig) {
+class AppConfig @Inject()(
+  config: Configuration,
+  servicesConfig: ServicesConfig,
+  environment: Environment,
+  @Named("appName") serviceIdentifier: String
+) {
   private val contactBaseUrl = servicesConfig.baseUrl("contact-frontend")
 
   private val assetsUrl = config.get[String]("assets.url")
-  private val serviceIdentifier = "MyService"
 
+  val runningAsDev: Boolean = {
```

Comment:
why  do we need this to run as a dev?

Reply:
Is it to send the user to the stride auth stub?

Reply:
When running locally e.g service manager it needs to redirect to `localhost:someport/path`
When running in dev/qa etc it needs to redirect to `/path` 
So we need to be able to tell it which to redirect to

Reply:
Sorry!!!! I should have approved the PR.
You could have delegated that to the config.
All good!